hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -397,6 +398,7 @@ function delete_kustomize_manifests() {
     echo "[INFO] (Attempt ${retry_count}) Executing kustomize build command"
     retry_count=$((retry_count+1))
     ${KUSTOMIZE} build ${WORK_DIR} > ${WORK_DIR}/kustomize-build-output.yaml && break
+    ${YQ} -i 'del( .metadata.creationTimestamp | select(. == "null") )' ${WORK_DIR}/kustomize-build-output.yaml
```

Comment:
Can we move this just above the kubectl apply command as there is a `break` statement

Reply:
Done